test/unit/providerSpec.js

```diff
@@ -160,12 +160,12 @@ describe('provider', function() {
     expect(successSpy2).not.toHaveBeenCalled();
     expect(errorSpy2).toHaveBeenCalled();
```

Comment:
I didnt touch this file.. but I guess something got changed or it is a false positive on github.